server/src/main/java/umm3601/user/User.java

```diff
@@ -3,9 +3,29 @@
 import org.mongojack.Id;
 import org.mongojack.ObjectId;
 
+// There are two examples of suppressing CheckStyle
```

Comment:
I tried to document when I was suppressing warnings so students would have a sense of when/why one might basically turn off the airbags.